community-plugins.json

```diff
@@ -4545,6 +4545,12 @@
         "repo": "phibr0/obsidian-commander"
     },
     {
+        "id": "obsidian-count-down",
+        "name": "Obsidian-Count-Down",
+        "author": "hamolicious",
+        "description": "Adds a count down to selected dates for easier timing and tracking of TODOs or events",
+        "repo": "hamolicious/Obsidian-Count-Down"
+    },
         "id": "obsidian-attendance",
```

Comment:
This is not valid JSON.
there is a { missing before this.

Reply:
fixed ✅